app/assets/stylesheets/components/_buttons.scss

```diff
@@ -26,7 +26,19 @@
   &:hover {
     background-color: darken($red, 6%);
   }
-}  
+}
+
+.eui-btn--light-blue {
```

Comment:
Alicia suggested I make the "View Loss Report" button a different blue color so I made this new button style. Just want to make sure adding this is okay